tempfile.js

```diff
@@ -0,0 +1,21 @@
+function Paginate(p, lang) {
```

Comment:
Do we need this file in this PR? Would you be so kind as to remove it, please? 🙂